packages/video_player_avplay/lib/video_player.dart

```diff
@@ -22,6 +22,12 @@ export 'src/closed_caption_file.dart';
 export 'src/drm_configs.dart';
 export 'src/tracks.dart';
 
+/// This will be used to set the ResumeTime when player restore.
+typedef ResumeTimeCallback = int Function();
+
+/// This will be used to set the RecreateMessage when player restore.
+typedef RecreateMessageCallback = DataSource Function();
```

Comment:
It would be better to make the names a bit clearer. Both callbacks are used for setRestoreData.

RestoreTimeCallback seems more appropriate because TimeCallback is the time to decide whether to restore or not.

Also, RestoreDataCallback seems more appropriate because the purpose of DataSource Function() is to pass the data to be used when restoring.